### PR TITLE
Create C4BB initial files for testing

### PR DESCRIPTION
**Pull request by CaitlinV39**
## Description
Create test files to load search parameters and profiles for C4BB and create resources to test

## Related issues
N/A

## Testing
Manual & touchstone

## FHIR Team Checklist
- [X] **Update the title** of the PR to be succinct and less than 50 characters
- [X] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- [X] Tag the PR with the type of update: **Bug**, **Dependencies**, **Enhancement**, or **New-Feature**
- [X] Tag the PR with **Azure API for FHIR** if this will release to the managed service
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Skip

### CODE REVIEW

**Comment by CaitlinV39** [comment]
@Ivanidzo4ka - Third times the charm? :)

**Review by Ivanidzo4ka — APPROVED** [review]

File: docs/rest/CB44/C4BB.http [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+content-type: {{contentType}}
+
+< ./SearchParameters/EOBType.txt
+
+### ExplanationOfBenefit - service-date
+POST {{fhirurl}}/SearchParameter
+Authorization: Bearer {{token}}
+content-type: {{contentType}}
+
+< ./SearchParameters/EOBServiceDate.txt
```
Comment (brendankowitz) [inline docs/rest/CB44/C4BB.http:44]: typo? EOBServiceDate.json

**Pull request merged**